lib/plugins/aws/lib/findDeployments.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+const MAX_CONCURRENT_S3_DOWNLOADS = Number(process.env.SLS_MAX_CONCURRENT_S3_DOWNLOADS) || 7;
```

Comment:
Do we need to introduce an option for this? Or could we go with a sane default?

(any extra option is BC surface, as well as documentation, so if we can avoid it that's best :) )